src/prefect/cli/auth.py

```diff
@@ -1,20 +1,29 @@
 import click
+import os
 import pendulum
 from click.exceptions import Abort
 from tabulate import tabulate
 
 from prefect import Client, config
 from prefect.exceptions import AuthorizationError, ClientError
 from prefect.cli.build_register import handle_terminal_error, TerminalError
+from prefect.backend import TenantView
 
 
 def check_override_auth_token():
     if config.cloud.get("auth_token"):
-        click.secho(
-            "Auth token has been set in the config. The CLI cannot be used to manage "
-            "your auth.",
-            fg="red",
-        )
+        if os.environ.get("PREFECT__CLOUD__AUTH_TOKEN"):
+            click.secho(
+                "Auth token has been set in the environment. The CLI cannot be used to "
+                "manage your auth. Unset the key with `unset PREFECT__CLOUD__AUTH_TOKEN`",
+                fg="red",
+            )
+        else:
+            click.secho(
+                "Auth token has been set in the config. The CLI cannot be used to "
+                "manage your auth. Remove the key from your configuration file.",
```

Comment:
can we specify the location of the configuration file here?

Reply:
Probably, but its path can be changed and I don't know if it's worth the extra effort -- they had to modify it to set this value. We also don't know if they've modified a config file that they shouldn't to change this value since the config is actually a stack of files combined.